tsconfig.base.json

```diff
@@ -20,8 +20,5 @@
       ],
       "akita-ng-fire/*": ["dist/akita-ng-fire/*"]
     }
-  },
-  "angularCompilerOptions": {
```

Comment:
Is @angular/fire dependent of Ivy ? As far as I remember, it's still consider experimental to build a lib with Ivy

Reply:
ivy is forced enabled in angular 12, this line was telling us it was turning it off, but in reality its now always on. It was flashing up warnings that we were attempting to turn it off so I simply removed this.